tmt/steps/execute/internal.py

```diff
@@ -50,13 +53,13 @@ class ExecuteInternal(tmt.steps.execute.ExecutePlugin):
     # Supported keys
     _keys = ["script", "interactive"]
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args: Any, **kwargs: Any):
         super().__init__(*args, **kwargs)
         self._previous_progress_message = ""
         self.scripts = SCRIPTS
 
     @classmethod
-    def options(cls, how=None):
+    def options(cls, how: Optional[str] = None) -> Any:
```

Comment:
Please, add TODO, this `Any` can be fixed later. I can't find the PR in which this was discussed, but we recently find out what the correct type should be.

Reply:
Unresolving this. The PR is #1304 and the method should return `List[ClickOptionDecoratorType]`, so this should be fixed once #1304 is merged.

Reply:
That PR has been merged, so I fixed the code accordingly